packages/amplication-data-service-generator/src/admin/entity/create-field-input.ts

```diff
@@ -55,9 +55,7 @@ const DATA_TYPE_TO_FIELD_INPUT: {
         <SelectArrayInput optionText={${relatedEntity.name}Title} />
       </ReferenceArrayInput>`;
     }
-    return jsxElement`<ReferenceInput source="${relatedEntity.name.toLowerCase()}.id" reference="${
-      relatedEntity.name
-    }" label="${field.displayName}">
+    return jsxElement`<ReferenceInput source="${field.name}.id" reference="${relatedEntity.name}" label="${field.displayName}">
```

Comment:
are you sure the `.id` is required here?

```suggestion
    return jsxElement`<ReferenceInput source="${field.name}" reference="${relatedEntity.name}" label="${field.displayName}">
```

Reply:
@yuval-hazaz why do you think that the `.id' can be removed?

Reply:
1. the source property on all other types is only using {field.name} without ID. 
2. field name can be "VIPCustomer" while the entity name can be "Customer" - so I am not sure if the usage of VIPCustomer.Id is correct. 

I am suggesting testing it carefully and making sure whether it is correct or not.